src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts

```diff
@@ -782,12 +782,12 @@
     <message>
         <location filename="../pages/getPage.cpp" line="55"/>
         <source>Sound Themeing</source>
-        <translation type="unfinished"></translation>
+        <translation>Ljud teman</translation>
     </message>
     <message>
         <location filename="../pages/getPage.cpp" line="55"/>
-        <source>Change basic sound settings</source>
-        <translation type="unfinished"></translation>
+        <source>Change grundläggande ljudinställningar</source>
```

Comment:
```suggestion
        <source>Change basic sound settings</source>
```
Do not change the "source" text - that could result in the translation not getting applied properly.